README.md

```diff
@@ -138,6 +138,18 @@ If you are using JRuby then you need to download latest [Oracle JDBC driver](htt
 * in `JRUBY_HOME/lib` directory
 * or include path to JDBC driver jar file in Java `CLASSPATH`
 
+Make sure to setup the following Oracle-specific environment variables properly
+
+* [NLS_LANG](http://www.orafaq.com/wiki/NLS_LANG) - preferred value `NLS_LANG=AMERICAN_AMERICA.AL32UTF8`
+* [ORA_SDTZ](http://docs.oracle.com/cd/E18283_01/server.112/e10729/ch4datetime.htm#CBBEEAFB) The setting should point a machine timezone like: `ORA_SDTZ=Europe/Riga`, otherwise Oracle by default uses a Fixed-offset timezone (like `03:00`) that is not daylight saving (DST) aware, which will lead to wrong translations of the timestamp values between Ruby code (DTS-aware) and Oracle session (non-DST-aware).  
+* [ORACLE_HOME](http://www.orafaq.com/wiki/ORACLE_HOME)
+
+You may either alter your environment settings or set the values in file `spec\support\custom_config.rb`. Sample file `custom_config.rb.sample` shows how to do that.
```

Comment:
Path separator should be `/` instead of `\` - in order to be consistent with other descriptions.